ecommerce/CheckoutController.php

```diff
@@ -187,8 +213,40 @@ private function prepare_items( $item_ids, $order_type = OrderModel::TYPE_SINGLE
 				$plan_info = apply_filters( 'tutor_get_plan_info', null, $item_id );
 				if ( $plan_info ) {
 					$item_name                   = $plan_info->plan_name;
```

Comment:
Why is the subscription's logic written in the core files when it's part of the add-on?

Reply:
We'll do it later